blazor/datagrid/paging.md

```diff
@@ -329,6 +329,75 @@ In the below sample, we have customized the Pager component in DataGrid control
 
 > You can refer to our [Blazor Grid Pager Template](https://blazor.syncfusion.com/demos/datagrid/pager-template) online demo of Pager Template feature in Blazor DataGrid.
 
+## How to render Pager at the top of the Grid
+
+By default, the Pager component will be rendered at the bottom of the Grid when the [AllowPaging](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_AllowPaging) property is enabled. To render the Pager component at the top of the Grid, the SfPager component is used.
```

Comment:
To render the Pager component at the top of the Grid, the SfPager component is used. - it is not understandable elaborate like, " Using SfPager component it is possible to render the Pager as top of the grid. For this we need to disable the default pager of Grid by setting AllowPaging as false and render the Sfpager externally. And further syncing the pagination action with grid by using the following way.